salt/files/minion.d/f_defaults.conf

```diff
@@ -1080,7 +1080,7 @@ ext_pillar:
 # the coloring of the messages, these color formatters also include padding as
 # well.  Color LogRecord attributes are only available for console logging.
 #
-{{ get_config('log_fmt_console', "'%(colorlevel)s %(colormsg)s'") }}
+#log_fmt_console "%(colorlevel)s %(colormsg)s"
```

Comment:
Line looks wrong too. I think 'message' should be 'colormsg' ?

Reply:
see other comment.